tst/providers/MeetingProvider/index.test.tsx

```diff
@@ -0,0 +1,30 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import '@testing-library/jest-dom';
+
+import { act, renderHook } from '@testing-library/react-hooks';
+import { LogLevel } from 'amazon-chime-sdk-js';
+import React from 'react';
+
+import { MeetingProvider, useMeetingManager } from '../../../src';
```

Comment:
We have used absolute paths over relative in this as well as JS SDK, could we import the same way we do in `MeetingManager` test? 

```
....from 'src/providers/MeetingProvider/MeetingManager'.
```

Reply:
Actually, pretty much all of the tests are using relative paths in React SDK. JS SDK is also using relative paths. https://github.com/aws/amazon-chime-sdk-js/blob/main/test/devicecontroller/DefaultDeviceController.test.ts#L8-L33

Reply:
Sorry my bad, I meant import like below:
```
import { MeetingProvider, useMeetingManager } from '../../../src/providers/MeetingProvider/MeetingManager';
```

Over 
```
import { MeetingProvider, useMeetingManager } from '../../../src';
```

Reply:
Done